### PR TITLE
fine-toothed comb pass, including addressing comments in #112

### PR DESCRIPTION
**Pull request by petrelharp**
I'm going through the whole thing - currently up to the Methods but will continue. Looks like we've got a conflicting PR in #126.

### CODE REVIEW

**Comment by petrelharp** [comment]
Done going through; will reconcile with #126 now.

**Comment by petrelharp** [comment]
I've rebased against #126; so this should apply cleanly after merging Adam's edits.

**Comment by andrewkern** [comment]
reviewing this now

File: popSimManu.tex [hunk 1]
```diff
@@ -122,26 +122,26 @@
...
 can be used to systematically compare demographic inference methods,
 and we encourage an even broader
-community of developers to join in the effort to standardize
-population genetic methods.}
+community of developers to contribute to this growing resource.
+}
 \date{}
 
-\keywords{Population genetics, Simulation, Inference}
+\keywords{Population genetics, Simulation, Inference, Reproducibility}
```
Comment (andrewkern) [inline popSimManu.tex:140]: should we add "Standardization" or "Standards"?

File: popSimManu.tex [hunk 2]
```diff
@@ -166,8 +165,8 @@ \section*{Introduction}
 methods using bespoke simulations. 
 Often these studies focus more on showcasing a novel method than on
 rigorously comparing it with competing methods.
-Moreover, this approach results in a great deal of duplicated effort across research
-groups, and contributes to decreased reproducibility and transparency.
+Moreover, this results in a great deal of duplicated effort,
```
Comment (andrewkern) [inline popSimManu.tex]: unclear to what "this" refers back to in this context. I'd keep it "this approach"

File: popSimManu.tex [hunk 3]
```diff
@@ -227,43 +227,44 @@ \section*{Introduction}
...
+% (\texttt{$\sim$/.local/cache/stdpopsim} by default on Unix platforms)
+where it can be quickly retrieved for subsequent simulations.
 In the initial version of \stdpopsim we support
 the HapMapII~\citep{international2007second} and
-deCODE~\citep{kong2010fine} genetic maps in humans;
-the \cite{salome2011recombination} map in \emph{A.~thaliana};
-and the \cite{comeron2012many} map in \emph{D.~melanogaster}.
+deCODE~\citep{kong2010fine} genetic maps for humans;
+the \cite{salome2011recombination} map for \emph{A.~thaliana};
+and the \cite{comeron2012many} map for \emph{D.~melanogaster}.
```
Comment (andrewkern) [inline popSimManu.tex:267]: maybe add a quick sentence here that says "Adding further maps into the resource is trivial"

File: popSimManu.tex [hunk 4]
```diff
@@ -277,16 +278,16 @@ \section*{Results}
...
 \citep{kelleher2016efficient,kelleher2018efficient}. Currently,
 \stdpopsim uses the  \texttt{msprime} coalescent simulator \citep{kelleher2016efficient}
-as the default simulation engine. We plan to include other simulation
-engines to support scenarios that cannot be modeled under the coalescent framework,
-and have already implemented \texttt{SLiM} \citep{haller2019tree,haller2019slim} as
-an additional backend.
+as the default simulation engine.
+We have implemented \texttt{SLiM} \citep{haller2019tree,haller2019slim} 
+as an alternative backend,
+to allow simulation of processes that cannot be modeled under the coalescent.
```
Comment (andrewkern) [inline popSimManu.tex:289]: what if we make this a positive thing and say "to allow simulation of processes that can only be modeled forward in time"?

File: popSimManu.tex [hunk 5]
```diff
@@ -364,37 +363,42 @@ \subsection*{The Species Catalog}
...
-isolation with migration (IM) model which can be used with any genome and genetic map combination.
+In addition to organism-specific models, \stdpopsim also includes a generic piecewise constant size model and
+isolation with migration (IM) model which can be used with any genome and genetic map.
 
-We note that we have put a good deal of effort into coming up with a standardized naming
+To guarantee reproducibility,
+we have standardized naming
 conventions for species, genetic maps, and demographic models that will enable long term stability
-of unique identifiers used throughout \stdpopsim. This is an essential feature to guarantee reproducibility.
+of unique identifiers used throughout \stdpopsim.
```
Comment (andrewkern) [inline popSimManu.tex]: should we give the readers a link to the documentation on this?

File: popSimManu.tex [hunk 6]
```diff
@@ -410,71 +414,75 @@ \subsection*{Use case: comparing methods of demographic inference}
 replicate human genomes under the OutOfAfricaArchaicAdmixture\_5R19 model and using the
 HapMapII\_GRCh37 genetic map~\citep{international2007second}. From top to bottom we show estimates for each
 of the three populations in the model (YRI, CEU, and CHB). In shades of blue we show the estimated
-$N(t)$ trajectories for each replicate. In black we show the true population size history as inferred
-for the rate of coalescence in the demographic model.}
+$N(t)$ trajectories for each replicate.
+As a proxy for the truth, in black we show inverse coalescence rates
+as calculated from the true demographic model (see text).
```
Comment (andrewkern) [inline popSimManu.tex:426]: see main text? see methods?

File: popSimManu.tex [hunk 7]
```diff
@@ -492,28 +500,28 @@ \subsection*{Use case: comparing methods of demographic inference}
...
 \end{center}
 \end{figure}
 
-\paragraph*{Multi-population demographic models}
-As \texttt{stdpopsim} implements multi-population demographic models, we also
+\paragraph*{Multi-population demographic models.}
+As \stdpopsim implements multi-population demographic models, we also
 explored parameter estimation of population divergence parameters. In particular,
-we simulated data under multi-population models for humans and \emph{Drosophila}
+we simulated data under multi-population models for humans and \textit{Drosophila}
```
Comment (andrewkern) [inline popSimManu.tex]: D. melanogaster for consistency

**Review by andrewkern — APPROVED** [review]
some small comments in here but one bigger thing we are missing: we don't have a link to the `stdpopsim` repo! I can add that in a separate PR if you are busy @petrelharp

File: popSimManu.tex [hunk 8]
```diff
@@ -659,42 +661,43 @@ \section*{Methods}
...
 As a consortium we have agreed to a standardized procedure for model inclusion
 into \stdpopsim that allows for rigorous quality control. Imagine Developer A
 wants to introduce a new model into \stdpopsim. Developer A implements the
 demographic model for the relevant organism along with clear documentation
-of the model parameters and populations. This model is submitted as a Pull Request,
+of the model parameters and populations. This model is submitted as a ``pull request'',
 where it is evaluated by a reviewer and then included as `preliminary',
 but is not linked to the online documentation nor the command line interface.
-Inclusion of a preliminary model triggers a QC issue, and a second developer,
+Inclusion of a preliminary model triggers a quality control (QC) issue, and a second developer,
```
Comment (agladstein) [inline popSimManu.tex]: I would make this active (since it's not automatic). (e.g "Developer A submits a quality control issue...")

**Review by agladstein — APPROVED** [review]
Lots of good edits here. I think it reads much better.

File: popSimManu.tex (hunk 1 above)
Comment (petrelharp) [reply popSimManu.tex:140]: is that a keyword? dunno.

File: popSimManu.tex (hunk 2 above)
Comment (petrelharp) [reply popSimManu.tex]: it's not an "approach"; that's the problem. Changing to "this situation".

File: popSimManu.tex (hunk 4 above)
Comment (petrelharp) [reply popSimManu.tex:289]: well, that's not quite the same thing? I vote to keep this.

File: popSimManu.tex (hunk 5 above)
Comment (petrelharp) [reply popSimManu.tex]: a bit unwieldy, but ok: https://stdpopsim.readthedocs.io/en/latest/development.html#naming-conventions

File: popSimManu.tex (hunk 6 above)
Comment (petrelharp) [reply popSimManu.tex:426]: I think this is OK.

**Comment by petrelharp** [comment]
I've addressed your comments above? I can squash if you like.

File: popSimManu.tex [hunk 9]
```diff
@@ -659,42 +665,43 @@ \section*{Methods}
...
 authors of the published model must be contacted
-to resolve ambiguities. This process can lead to cycles of revision of
-the implementation from Developers A and B but allows the project to assure
-quality. Indeed we have already had a number of interesting cases of arbitration
-to this point. Further details of our QC process can be found in our
+to resolve ambiguities. 
+% This process can lead to cycles of revision of the implementation from Developers A and B but allows the project to assure quality.
+% Indeed we have already had a number of interesting cases of arbitration to this point. 
+Further details of our QC process can be found in our
 \href{https://stdpopsim.readthedocs.io/en/latest/development.html#}{developer documentation}.
```
Comment (agladstein) [inline popSimManu.tex]: This hyper ref is not displaying the link.
Screen Shot 2019-12-20 at 1 15 17 PM

Comment (petrelharp) [reply popSimManu.tex]: whoops; meant to do `url`

**Review by agladstein — APPROVED** [review]

**Comment by petrelharp** [comment]
Here's a squash. Should be ready to merge after #126.

**Pull request merged**